projects/sandbox/src/app/progress-tracker/progress-tracker.component.html

```diff
@@ -186,4 +196,93 @@ <h2 class="title-id">{{ datasetId }}</h2>
       </span>
     </ng-container>
   </div>
+  <div
+    class="warning-view"
+    *ngIf="getWarningViewCount(); let warningViewCount"
+    [ngClass]="{
+      closed: !warningViewOpen
+    }"
+  >
+    <sb-navigation-orbs
+      libClickAware
+      [ignoreClasses]="['link-internal', 'nav-orb', 'warning-view-content']"
+      [clickAwareIgnoreWhen]="!showing"
+      (clickOutside)="closeWarningView()"
+      ariaLabel="Tier Zero Warnings Navigation"
+      class="sb-navigation-orbs vertical zero-warning"
+      [count]="warningViewCount"
+      [fnClassMapInner]="getOrbConfigInner.bind(this)"
+      [tooltips]="[
+        'content tier 0 records found - click to see samples',
+        'media tier 0 records found - click to see samples'
+      ]"
+      (clickEvent)="setWarningView($event)"
+    ></sb-navigation-orbs>
+    <div
+      class="warning-view-content"
+      libClickAware
+      [ignoreClasses]="['link-internal', 'nav-orb', 'sb-navigation-orbs']"
+      [clickAwareIgnoreWhen]="!showing"
+      (clickOutside)="closeWarningView()"
+    >
+      <ng-template #warningViewHeader let-list="list" let-label="label">
+        <span class="warning-view-header">
+          This dataset contains
+          {{
+            list.length | i18nPlural: { '=1': 'one record that was', other: '# records that were' }
+          }}
+          awarded {{ label }} 0.
+          <ng-container *ngIf="list.length > 1">
+            Here follows a sample list of records
+          </ng-container>
+        </span>
+      </ng-template>
+      <ng-container *ngIf="getZeroTierLinks('content-tier'); let zeroTierContentLinks">
+        <div *ngIf="warningDisplayedTier === DisplayedTier.CONTENT">
+          <a (click)="closeWarningView()" class="warning-animated warning-view-title"
+            >Content Tier Zero ({{ zeroTierContentLinks.length }})</a
```

Comment:
If not mistaken, this is the length of the samples list, we want the value from field 'total'

Reply:
Fixed